src/components/button/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { default } from './button';
```

Comment:
I think its better to group our components by their atomic groups would you agree? So like instead of src/components/button/index.tsx it would be something like src/components/atoms/button/index.tsx

Reply:
Okay i reupdate on this .. thanks!